examples/controlnet/train_controlnet_sd3.py

```diff
@@ -655,6 +694,7 @@ def make_train_dataset(args, tokenizer_one, tokenizer_two, tokenizer_three, acce
             dataset = load_dataset(
                 args.train_data_dir,
                 cache_dir=args.cache_dir,
+                trust_remote_code=True,
```

Comment:
Nice, could be an argument but more convenient like this especially as the example dataset requires it. Can be replicated across training scripts.
For future reference we should look at the `num_proc` option which should help speed up processing.